homeassistant/components/tesla_fleet/icons.json

```diff
@@ -185,6 +185,18 @@
       },
       "wall_connector_state": {
         "default": "mdi:ev-station"
+      },
+      "storm_mode_active": {
+        "default": "mdi:weather-lightning"
+      },
+      "island_status": {
+        "default": "mdi:help-circle",
+        "state": {
+          "on_grid": "mdi:transmission-tower",
+          "off_grid": "mdi:transmission-tower-off",
+          "island_status_unknown": "mdi:help-circle",
+          "off_grid_intentional": "mdi:account-cancel"
```

Comment:
Can you also add off_grid_unintentional, as I've had a user report it's also a valid state.

Reply:
Its already there, in line 198.

Reply:
Fixed in [a682d25](https://github.com/home-assistant/core/pull/126438/commits/a682d2504606bbb38b4dfe8fe27fea06a36a81ea).